apps/expo-go/android/expoview/src/main/java/host/exp/exponent/LauncherActivity.kt

```diff
@@ -70,6 +66,26 @@ class LauncherActivity : Activity() {
     )
   }
 
+  override fun onResume() {
+    super.onResume()
+    isAppInForeground = true
+    startStayAwakeServiceIfNeeded()
+  }
+
+  override fun onPause() {
+    super.onPause()
+    isAppInForeground = false
+  }
+
+  private fun startStayAwakeServiceIfNeeded() {
+    if (isAppInForeground) {
```

Comment:
Do we need to this check? It looks like this function is only called in the `onResume` function, so should it always be true?

Reply:
Probably not, might be over cautious. We could just drop the variable and always call the function in `onResume`

Reply:
Yeah, we'll do that and I'll keep an eye on crashlytics to see if it still happens. It shouldn't